tools/webhook_test.py

```diff
@@ -125,6 +145,46 @@ def int_or_default(input_parm):
         raw_input(), payload["message"][input_parm])
 
 
+def get_gym_info(gym_id):
+    """ Gets the information about the gym. """
+    return _gym_info.get(gym_id, _default_gym_info)
+
+
+def check_gym(questionable_input):
+    if get_gym_info(questionable_input)['name'] != "unknown":
```

Comment:
Nobody will be able to get the gym id without jumping through a bunch of hoops.
This would probably be better as presenting as a list gym_name options and allowing the user to pick one to make it easier on the user.
I'd imagine nobody wants to remember a gym id :smile:

Reply:
This won't be practical if there are alot of gyms in the users cache,